Users/pracheeti/helloworld (#2)
FRC5190/2020Training: kotlin/hello_world.md

# Hello world

> In this project, you will write your first kotlin program that will print Hello World.

Reviewer: ```suggestion
In this project, you will write your first Kotlin program that will print `Hello World`.
```

## Prerequisites

None

## Instructions

> 1. Install Java on your Windows or MAC laptop following the instructions [here](https://www.oracle.com/technetwork/java/javase/downloads/index.html).

Reviewer: ```suggestion
1. Install Java on your Windows, Mac, or Linux computer following the instructions [here](https://www.oracle.com/technetwork/java/javase/downloads/index.html).
```

2. Install IntelliJ using the instructions [here](https://www.jetbrains.com/idea/download/#section=windows).

## Steps

> 1. Create a new Kotlin JVM project in IntelliJ:  File -> New -> Project Select. In the project SDK dropdown, select the java verison that you installed in step 2 of the instructions. Select Kotlin/JVM under Additional Libraries and Framework. Press Create. Name your project and continue.

Reviewer: ```suggestion
1. Create a new Kotlin JVM project in IntelliJ:  `File` ->` New` -> `Project Select`. In the project SDK dropdown, select the Java verison that you installed in step 2 of the instructions. Select `Kotlin/JVM` under `Additional Libraries and Framework`. Press `Create`. Name your project and continue.
```

> 2. Once your project loads, click the dropdown on your project folder. Right click the src folder and select 

Reviewer: ```suggestion
2. Once your project loads, click the dropdown on your project folder. Right click the `src` folder and select

>    New -> Kotlin File/Class. Name your file HelloWorld.

Reviewer: ```suggestion
`New` -> `Kotlin File/Class`. Name your file `HelloWorld`

> 3. Write a method called main. The body of the method should print "Hello World" onto the console when the program is run.

Reviewer: ```suggestion
3. Write a method called `main`. The body of the method should print "Hello World" onto the console when the program is run.
```
